.github/workflows/mdlint.yml

```diff
@@ -0,0 +1,20 @@
+name: markdownlint
+on:
+  push:
+    paths:
+      - '**/*.md'
+      - '.markdownlint.json'
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  mdlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
`checkout@v4` to avoid NodeJS deprecation issue

Reply:
ipynb and py files :/

Reply:
According to the doc, you can put a `.markdownlintignore` file at the root of the repo to list the things to ignore (similar syntax at the `.gitignore` one). As that's processed by the underlying tool and not the action itself, it may work more reliably.

Reply:
For some reason the `.markdownlintignore` refuses to ignore `README.md` (at least it ignores ipynb and py). But these two lines are a mistake:

Reply:
They are 253 characters long though and could be shortened but would reduce the readability of the table.
Probably some tweaks on the configuration file. This can wait a following PR as we're also missing the custom matcher to annotate the PR.